### ISSUE TITLE
why is the data in folder hypotheses abnormal ?

### ISSUE THREAD

**Issue by KelleyYin** [microsoft/MASS#8]
In my code, both the valid and test set are normal. 
```
INFO - 06/26/19 16:40:35 - 0:00:07 - ============ Data summary
INFO - 06/26/19 16:40:35 - 0:00:07 - Monolingual data   - train -           en:   5000000
INFO - 06/26/19 16:40:35 - 0:00:07 - Monolingual data   - valid -           en:      3000
INFO - 06/26/19 16:40:35 - 0:00:07 - Monolingual data   -  test -           en:      2999
INFO - 06/26/19 16:40:35 - 0:00:07 - Monolingual data   - train -           de:   5000000
INFO - 06/26/19 16:40:35 - 0:00:07 - Monolingual data   - valid -           de:      3000
INFO - 06/26/19 16:40:35 - 0:00:07 - Monolingual data   -  test -           de:      2999
INFO - 06/26/19 16:40:35 - 0:00:07 - Parallel data      - valid -        de-en:      3000
INFO - 06/26/19 16:40:35 - 0:00:07 - Parallel data      -  test -        de-en:      2999
``` 
However, why is the data in folder `hypotheses`  abnormal ?
```
$ wc -l  hypotheses/*
   50 hyp0.de-en.test.txt
   50 hyp0.de-en.valid.txt
   50 hyp0.en-de.test.txt
   50 hyp0.en-de.valid.txt
   50 hyp1.de-en.test.txt
   50 hyp1.de-en.valid.txt
   50 hyp1.en-de.test.txt
   50 hyp1.en-de.valid.txt
   50 ref.de-en.test.txt
   50 ref.de-en.valid.txt
   50 ref.en-de.test.txt
   50 ref.en-de.valid.txt

```
Why do they only have 50 sentences ?

**Comment by StillKeepTry** [comment]
@KelleyYin Sorry, it is a bug which upload code for supporting english-only tasks. It has been fixed. Can you try again?

**Comment by KelleyYin** [comment]
@StillKeepTry https://github.com/microsoft/MASS/blob/bfee003b89c526f26b3ed4a2737a566ae19a5f06/MASS/src/model/transformer.py#L266
If you reload the pre-trained model , the language embedding of encoder will not be created here.

**Comment by StillKeepTry** [comment]
Do you set `english-only` as true? If this flag is set as true, which means only using one language data, and of course it does not need language embedding any more.

The part of english-only task is still under testing. I will complete and upgrade this function  as soon as possible. Before that, you can try NMT tasks first (`english-only` should be set as false in NMT tasks).

**Comment by KelleyYin** [comment]
@StillKeepTry Yes, I did't notice the flag `english_only` before. I will set the flag to False by default.
